third_party/js.bzl

```diff
@@ -17,437 +17,473 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+                "https://raw.githubusercontent.com/catapult-project/catapult/2f7ee994984f3ebd3dd3dc3e05777bf180ec2ee8/LICENSE",
+            ],
+            "b1f0195f305ca66fdb7dae264771f162ae03f04aa642848f15cd871c043e04d1": [
+                "https://mirror.bazel.build/raw.githubusercontent.com/catapult-project/catapult/237aea8b58a37a2991318b6a0db60d84078e5f7e/trace_viewer_full.html",
+                "https://raw.githubusercontent.com/catapult-project/catapult/237aea8b58a37a2991318b6a0db60d84078e5f7e/trace_viewer_full.html",  # 2017-06-19
+            ],
+        },
+    )
 
-  filegroup_external(
-      name = "org_d3js",
-      # no @license header
-      licenses = ["notice"],  # BSD-3-Clause
-      sha256_urls_extract = {
-          "05a9c2b9c206447be0e26b3a705e7f8df4943df2d063ddc5bf0274f50ec44727": [
-              "https://mirror.bazel.build/github.com/d3/d3/releases/download/v5.7.0/d3.zip",
-              "https://github.com/d3/d3/releases/download/v5.7.0/d3.zip",
-          ],
-      },
-      # TODO(jart): Use srcs=["d3.js"] instead of this once supported.
-      generated_rule_name = "all_files",
-      extra_build_file_content = "\n".join([
-          "filegroup(",
-          "    name = \"org_d3js\",",
-          "    srcs = [\"d3.js\"],",
-          ")",
-      ]),
-  )
+    http_archive(
+        name = "ai_google_pair_facets",
+        sha256 = "e3f7b7b3c194c1772d16bdc8b348716c0da59a51daa03ef4503cf06c073caafc",
+        strip_prefix = "facets-0.2.1",
+        urls = [
+            "http://mirror.bazel.build/github.com/pair-code/facets/archive/0.2.1.tar.gz",
+            "https://github.com/pair-code/facets/archive/0.2.1.tar.gz",
+        ],
+    )
 
-  filegroup_external(
-      name = "org_chromium_catapult_vulcanized_trace_viewer",
-      licenses = ["notice"],  # BSD-3-Clause
-      sha256_urls = {
-          "f0df289ba9d03d857ad1c2f5918861376b1510b71588ffc60eff5c7a7bfedb09": [
-              "https://mirror.bazel.build/raw.githubusercontent.com/catapult-project/catapult/2f7ee994984f3ebd3dd3dc3e05777bf180ec2ee8/LICENSE",
-              "https://raw.githubusercontent.com/catapult-project/catapult/2f7ee994984f3ebd3dd3dc3e05777bf180ec2ee8/LICENSE",
-          ],
-          "b1f0195f305ca66fdb7dae264771f162ae03f04aa642848f15cd871c043e04d1": [
-              "https://mirror.bazel.build/raw.githubusercontent.com/catapult-project/catapult/237aea8b58a37a2991318b6a0db60d84078e5f7e/trace_viewer_full.html",
-              "https://raw.githubusercontent.com/catapult-project/catapult/237aea8b58a37a2991318b6a0db60d84078e5f7e/trace_viewer_full.html"  # 2017-06-19
-          ],
-      },
-  )
+    web_library_external(
+        name = "vaadin_vaadin_split_layout",
+        licenses = ["notice"],  # Apache License 2.0
+        sha256 = "44fb83628edb77cb8392c165d4d99734750a6fbb00e5391f033962e56f14eba3",
+        urls = [
+            "https://github.com/vaadin/vaadin-split-layout/archive/v1.1.0.tar.gz",
```

Comment:
please add an entry for mirror. It should read like
`"https://mirror.bazel.build/github.com/vaadin/vaadin-split-layout/archive/v1.1.0.tar.gz"`

here and below.

Reply:
Done